src/plugins/telemetryTable/components/TableComponent.vue

```diff
@@ -569,11 +572,34 @@ export default {
     this.table.destroy();
   },
   methods: {
-    setLoadingPromise() {
-      this.loadFinishResolve = null;
-      this.isFinishedLoading = new Promise((resolve, reject) => {
-        this.loadFinishResolve = resolve;
-      });
+    addToAfterLoadActions(func) {
+      this.afterLoadActions.push(func);
+    },
+    runAfterLoadActions() {
+      if (this.afterLoadActions.length > 0) {
+        this.afterLoadActions.forEach((action) => action());
+        this.afterLoadActions = [];
+      }
+    },
+    handleConfigurationChanges(changes) {
+      const { rowLimit, telemetryMode, persistModeChange } = changes;
+
+      this.persistModeChange = persistModeChange;
+
+      if (this.rowLimit !== rowLimit) {
```

Comment:
@akhenry this was previously `this.rowLimit === rowLimit` which was causing duplicate unsubscribes with the second unsubscribe failing because it was already removed in the realtime-provider.